.rubocop.yml

```diff
@@ -1,6 +1,7 @@
 require:
   - rubocop-rspec
   - rubocop-performance
+  - rubocop-packaging
```

Comment:
I've never heard of this gem. Why do we need it here? Is it a new rubocop addition?

Reply:
`rubocop-packaging` is an extension of RuboCop ([list of known extensions here](https://docs.rubocop.org/rubocop/extensions.html)).
rubocop-packaging was born because while maintaining Ruby gems downstream (in Debian), we faced some difficulties like upstream using `git` in `gemspec`, or using relative-path from `spec` to `lib` and etcetera. It helps to enforce some of the guidelines so that the downstream can build their packages in a clean environment without any problems. 
It's more like an upstream-downstream co-operation and collaboration! :smile: 

We've started raising these problems amongst upstream maintainers and libraries and they're so far welcoming this change (since it's a win-win! :D)

Please let me know if I should describe it more or if you'd want me to explain anything, I'll be more than happy to! :smile:

Reply:
Seems ok for now I guess.